### PR TITLE
Add privatelink property support to processing of WebDAV responses

### PR DESCRIPTION
**Pull request by davivel**
Do not merge before release 2.5.0 of app.

### CODE REVIEW

File: src/com/owncloud/android/lib/resources/files/ReadRemoteFolderOperation.java [hunk 1]
```diff
@@ -171,6 +171,7 @@ private RemoteFile fillOCFile(WebdavEntry we) {
         file.setSize(we.size());
         file.setQuotaUsedBytes(we.quotaUsedBytes());
         file.setQuotaAvailableBytes(we.quotaAvailableBytes());
+        file.setPrivateLink(we.privateLink());
```
Comment (davigonz) [inline src/com/owncloud/android/lib/resources/files/ReadRemoteFolderOperation.java]: Could you please rename this variable **we** as **webdavEntry**? Clean code philosophy, you know...

File: src/com/owncloud/android/lib/resources/files/RemoteFile.java [hunk 2]
```diff
@@ -33,132 +33,143 @@
...
+ * @author masensio
  */
 
 public class RemoteFile implements Parcelable, Serializable {
 
-    /** Generated - should be refreshed every time the class changes!! */
+    /**
+     * Generated - should be refreshed every time the class changes!!
+     */
     private static final long serialVersionUID = 3130865437811248451L;
```
Comment (davigonz) [inline src/com/owncloud/android/lib/resources/files/RemoteFile.java]: As the comment above says: "it should be refreshed every time the class changes" but has not been updated here

File: src/com/owncloud/android/lib/resources/files/RemoteFile.java [hunk 3]
```diff
@@ -33,132 +33,143 @@
...
         if (path == null || path.length() <= 0 || !path.startsWith(FileUtils.PATH_SEPARATOR)) {
             throw new IllegalArgumentException("Trying to create a OCFile with a non valid remote path: " + path);
         }
         mRemotePath = path;
-	}
-	
-	public RemoteFile(WebdavEntry we) {
+    }
+
+    public RemoteFile(WebdavEntry we) {
```
Comment (davigonz) [inline src/com/owncloud/android/lib/resources/files/RemoteFile.java]: Rename this variable as **webdavEntry** please

**Comment by davigonz** [comment]
@davivel could you have a look at the requested changes? Thanks

File: src/com/owncloud/android/lib/resources/files/ReadRemoteFolderOperation.java (hunk 1 above)
Comment (davigonz) [reply src/com/owncloud/android/lib/resources/files/ReadRemoteFolderOperation.java]: Done

**Comment by davivel** [comment]
@davigonz, sorry for the lack of response :(

**Pull request merged**